src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,14 @@
+import { UserInfo } from '../UserInfo';
+
 // Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+export const TodoInfo = ({ todo }) => (
+  <>
```

Comment:
There is no need to use fragments while you are rendering just one element inside